### ISSUE TITLE
Replace sort caret with Wātea styling

### ISSUE THREAD

**Issue by brynwhyman** [silverstripe/silverstripe-ckan-registry#169]
The frontend table is using a sorting icon provided by Griddle.

This should be:
- [ ]  Replaced with the caret provided with the Wātea theme
- [ ] Ensure the icon is always present on the column heading to highlight that the column is sort-able. (Griddle won't show the icon until the column heading is clicked)

**Comment by robbieaverill** [comment]
> Replaced with the caret provided with the Wātea theme

We should be careful not to couple it though. The theme isn't a dependency of this module.

**Comment by ScopeyNZ** [comment]
Agreed. This issue should probably read "use default SS styling" or something. Preferably Wātea should not require any updates for this.
